### PR TITLE
LPS-170477 DM some actions from ellipsis icon don't work

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Issue https://issues.liferay.com/browse/LPS-170477

Also fixed some warnings that was shown in browser console

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/DLPortlet.java [hunk 1]
```diff
@@ -41,8 +41,6 @@
 		"com.liferay.portlet.css-class-wrapper=portlet-document-library",
 		"com.liferay.portlet.display-category=category.cms",
 		"com.liferay.portlet.display-category=category.highlighted",
-		"com.liferay.portlet.footer-portlet-javascript=/document_library/js/legacy/main.js",
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/DLPortlet.java:44]: The messages were annoying!

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:stop

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**